daemon/daemon.go

```diff
@@ -322,10 +327,24 @@ func (d *Daemon) writeNetdevHeader(dir string) error {
 
 	fw := bufio.NewWriter(f)
 	fw.WriteString(d.conf.Opts.GetFmtList())
+	fw.WriteString(d.fmtPolicyEnforcement())
 
 	return fw.Flush()
 }
 
+// returns #define for PolicyEnforcement based on the configuration of the daemon.
+// Must be called with d.conf.EnablePolicyMU locked.
+func (d *Daemon) fmtPolicyEnforcement() string {
+	d.GetPolicyRepository().Mutex.RLock()
+	enforcement := d.EnablePolicyEnforcement()
```

Comment:
Note: this always returns false in the case of when Cilium is running in tandem with K8s in 'default' mode, meaning that we always allow egress traffic to the host. Discussed offline with @tgraf , and we both agree that this isn't trivial.